src/game/engine/abstract.cpp

```diff
@@ -54,3 +54,24 @@ int AbstractClass::Distance_To_Target(target_t target)
 
     return distance;
 }
+
```

Comment:
All these single liners should really just be left in the header so they get inlined.